modules/manage/pages/kubernetes/k-manage-resources.adoc

```diff
@@ -26,7 +26,9 @@ kubectl describe nodes
 
 == Limitations
 
-Redpanda does not support decreasing the CPU cores for brokers in an existing cluster.
+Decreasing the number of CPU cores in a production cluster is not supported in this version of Redpanda. However, you can enable xref:manage:cluster-maintenance/cluster-balancing.adoc#intra-broker-partition-balancing[intra-broker partition balancing] to reduce CPU cores in a development cluster.
+
+Starting from version 24.3, decreasing the number of CPU cores in a production cluster is supported. xref:24.3@ROOT:upgrade:index.adoc[Upgrade to version 24.3] or later to access this feature.
```

Comment:
@daisukebe does this phrasing sound good to you?

Reply:
Yes, sounds good and correct.